package.json

```diff
@@ -37,7 +37,10 @@
   "dependencies": {
     "extend": "^3.0.0",
     "findup-sync": "^0.4.1",
```

Comment:
@sttk are we going to be able to drop the `findup-sync` usage and replace with `fined`?